components/automate-deployment/pkg/server/server.go

```diff
@@ -1161,6 +1161,12 @@ func (s *server) shouldFetchManifest() bool {
 // the configured channel. If neither are available, an error is
 // returned.
 func (s *server) nextManifest() (*manifest.A2, error) {
+
+	if airgap.AirgapInUse() && s.deployment.Config.Deployment.GetV1().GetSvc().GetUpgradeStrategy().GetValue() != "none" {
+		logrus.Error("Airgapped deployments require the configuration setting upgrade_strategy=\"none\"")
+		return nil, errors.New("Airgapped deployments require the configuration setting upgrade_strategy=\"none\"")
```

Comment:
will this break people?
why does it have to be set to none again?

Reply:
The upgrade_strategy should be "none" for an airgapped installation so that the converger won't keep trying to reach the Internet to see if there is a new manifest available.

The people this might break are the ones using an airgap bundle to deploy when they aren't actually behind an airgap; they've been taking upgrades when they shouldn't have been able to. People using airgap bundles behind an airgap won't be worse off, and they might be better off because they'll get an error in the logs that says what to fix instead of a bunch of timeouts.

I think if enough things break, people using airgap bundles without being behind an airgap who take an upgrade containing this code could end up in trouble, so I will do something to write an error to the logs without throwing an actual error.

Reply:
i'm not sure if this is correct. airgap has it's own manifest provider that will only look at what is on disk. Can you double check

Reply:
You're right, and this case is a non-issue. I'll revert the change and update the PR.